components/ui-api-layer/internal/domain/servicecatalog/clusterserviceplan_converter.go

```diff
@@ -91,12 +97,28 @@ func (p *clusterServicePlanConverter) unpackCreateParameterSchema(raw []byte) (g
 }
 
 func (p *clusterServicePlanConverter) extractCreateSchema(raw []byte) (map[string]interface{}, error) {
+	raw = bytes.Trim(raw, "\x00")
```

Comment:
Hmm, is this necessary?

Reply:
Yes, this is necessary, but only for our unit tests, because we use standard implementation of base64, that sometimes add `\x00` chars at the end of bytes, when decode it some data.

Reply:
In fact, the null characters are added because of that: `base64.StdEncoding.DecodedLen(len(raw))`
It returns the max length of the encoded string. I would rather move that line to `unpackCreateParameterSchema`

https://golang.org/pkg/encoding/base64/#Encoding.DecodedLen

We need to trim that null characters, because `json.Unmarshal` is failing.